.githooks/pre-commit

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo
```

Comment:
`-o` doesn't do anything if you don't specify an option

```suggestion
set -euo pipefail
```

Reply:
There are no pipes but if you like it.